lib/scan.js

```diff
@@ -73,3 +79,29 @@ module.exports = async (rc = {}) => {
 
   return results.length ? 1 : 0
 }
+
+function canBeConfigured (module) {
+  return !!module.configSchema
+}
+
+/**
+ * Not only validates the config, but also removes fields that are not defined in schema and
+ * puts default value in if property is undefined
+ */
+function validateAndNormalizeModuleConfig (module, config) {
+  const ajv = new Ajv({
+    strictDefaults: true,
+    strictKeywords: true,
+    removeAdditional: true,
+    useDefaults: true
+  })
```

Comment:
`removeAdditional` - remove additional (nondocumented) properties from config object. Might be useful to force module developers documenting all possible config values.
`useDefaults` - if property is not defined and has default value will modify config object by injecting the property with default value. Just handy.